scripts/patrol-local-integration-test-with-docker.sh

```
@@ -1,40 +1,36 @@
-# Open android emulator
+#   - patrol_cli installed: dart pub global activate patrol_cli 4.3.1
+#   - ADB installed (Android SDK platform-tools)
+#   - A local Android emulator running (e.g. via Android Studio AVD)
+#
+# Usage:
+#   ./scripts/patrol-local-integration-test-with-docker.sh
 
-# Stoping previous environment if any
-killall ngrok || true
+# Stop previous backend environment if any
 cd backend-docker
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)